src/main/java/com/codecool/virtualpub/logic/Game.java

```diff
@@ -1,36 +1,86 @@
 package com.codecool.virtualpub.logic;
 
-import com.codecool.virtualpub.data.Bartender;
-import com.codecool.virtualpub.data.Customer;
-import com.codecool.virtualpub.data.Drink;
-import com.codecool.virtualpub.data.GameState;
+import com.codecool.virtualpub.data.*;
+import com.codecool.virtualpub.ui.Display;
+import com.codecool.virtualpub.ui.Input;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Game {
-    public void Game() {
+
+    private Input input = new Input();
+    private Display display = new Display();
+
+    public void run() {
         ArrayList<Customer> customers = new ArrayList<Customer>();
 
-        for (int i = 0; i < 10; i++) {
-            String name = "random bela";
-            customers.add(new Customer(name));
-        }
+        generateCustomers(customers);
```

Comment:
Why does generateCustomer need an empty arraylist to function? This just mutates the list that it receives. It could create its own list and return with it